backend/README.md

```diff
@@ -0,0 +1,25 @@
+# TrackYourBedBackend
+
+### Build docker image
+Run `package.sh` to create docker image called "trackyourbed", which runs the service. Consult *src/main/resources/application.yml* for the environment variables that need to be passed to the container to configure the database connection.
+
+### Initial setup of database
+As a superuser on the database instance, run
+```
+create user betten;
+alter role "betten" with login;
+alter user betten with password '<password>';
+create database betten;
+\connect betten
+create extension "uuid-ossp"
+```
```

Comment:
with the flyway integration, this seems to be obsolete.

Reply:
No, flyway acts on the initialized betten database